bots/minecraft.bot.js

```diff
@@ -0,0 +1,69 @@
+const _ = require('lodash');
+const Config = require('pico-conf');
+const Slack = require('pico-slack');
+const RCon = require('rcon');
+
+const ASKING_FOR_NOW = /(?:any\s*one|any\s*body).*(?:on|playing).*/i;
+const ASKING_FOR_LATER = /(?:when|any\s*one|any\s*body).*(?:planning|going|thinking).*(?:be on|playing|hopping on).*/i;
+
+Slack.onMessage(async (msg) => {
+	if (msg.channel != 'minecraft' && !msg.isDirect) {
```

Comment:
Isn't `msg.isDirect` only true if you're DMing Higgins?

Reply:
Yes, this is so that I can test it first my DMing Higgins before revealing it in the channel.